graf3d/eve7/src/REveDataCollection.cxx

```diff
@@ -118,52 +129,53 @@ Bool_t REveDataItemList::SetRnrState(Bool_t iRnrSelf)
    Bool_t ret = REveElement::SetRnrState(iRnrSelf);
    std::vector<int> ids;
 
-   for (size_t i = 0; i < fItems.size(); ++i) {
+   for (size_t i = 0; i < fItems.size(); ++i ) {
       ids.push_back(i);
       fItems[i]->SetRnrSelf(fRnrSelf);
    }
 
-   _handler_items_change(this, ids);
+   _handler_items_change( this , ids);
    StampVisibility();
    StampObjProps();
 
    return ret;
 }
 
 //______________________________________________________________________________
-void REveDataItemList::ProcessSelection(ElementId_t selectionId, bool multi, bool secondary,
-                                        const std::set<int> &secondary_idcs)
+void REveDataItemList::ProcessSelection(ElementId_t selectionId, bool multi, bool secondary, const std::set<int>& secondary_idcs)
 {
    RefSelectedSet() = secondary_idcs;
-   REveSelection *selection = (REveSelection *)ROOT::Experimental::gEve->FindElementById(selectionId);
+   REveSelection* selection = (REveSelection*) ROOT::Experimental::gEve->FindElementById(selectionId);
```

Comment:
Same clang-format problem here.
Also when doing explicit cast, one should better use `static_cast<REveSelection *>()`